lib/pbench/test/unit/server/test_schema.py

```diff
@@ -35,11 +36,11 @@ def test_enum(self):
             (ParamType.USER, "drb", "drb"),
         ),
     )
-    def test_successful_conversions(self, test, monkeypatch):
-        def ok(user: str) -> str:
-            return user
+    def test_successful_conversions(self, client, test, monkeypatch, pbench_token):
+        def ok(user: User, verified: bool) -> str:
+            return user, verified
 
-        monkeypatch.setattr(Auth, "validate_user", ok)
+        monkeypatch.setattr(Auth, "verify_user", ok)
```

Comment:
`verify_user` is an object method of the Auth class so I can't set the monkeypatch attribute as `monkeypatch.setattr(Auth(), "verify_user", ok)`. if `verify_user` is a static method then it would have been fine passing just one parameter, but since its an object method I have to pass 2 parameters (I think it considers `self` as 1st parameter).


But I think you pointed out a valid mistake, the function should return a User and a bool (in this case always `True`) and I should make the analogous change on line 62

So I think the right way to do this would be
```
def ok(user: User, username: str) -> (User, bool):
    return user, True

monkeypatch.setattr(Auth, "verify_user", ok)
```